generate-configs.py

```diff
@@ -0,0 +1,129 @@
+                        ],
+                        "measurement": f"MijiaBridge_{mac}_{reading_type}",
+                        "orderByTime": "ASC",
+                        "policy": "default",
+                        "resultFormat": "time_series",
+                        "select": [
+                            [
+                                {"params": ["value"], "type": "field"},
+                                {"params": [], "type": "mean"},
+                            ]
+                        ],
+                        "tags": [],
+                    }
+                )
+
+
+def mutate_smarthome_model(model, mac_to_name):
+    for mac, name in mac_to_name.items():
+        if f"MijiaBridge_{mac}" not in model:
+            print(f"adding MijiaBridge_{mac} => {name}")
+            model[f"MijiaBridge_{mac}"] = {
+                "class": "org.eclipse.smarthome.core.items.ManagedItemProvider$PersistedItem",
+                "value": {
+                    "groupNames": ["MijiaBridge"],
+                    "itemType": "Group",
+                    "tags": [],
+                    "label": f"{name} sensor",
+                },
+            }
+        for key_suffix, label_suffix in [
+            ("Temperature", "temperature"),
+            ("Humidity", "humidity"),
+            ("BatteryLevel", "battery level"),
+        ]:
+            key = f"MijiaBridge_{mac}_{key_suffix}"
+            value = {
+                "class": "org.eclipse.smarthome.core.items.ManagedItemProvider$PersistedItem",
+                "value": {
+                    "groupNames": [f"MijiaBridge_{mac}"],
+                    "itemType": "Number",
+                    "tags": [],
+                    "label": f"{name} {label_suffix}",
+                },
+            }
+            if key not in model:
+                print(f"adding {key} => {name}")
+                model[key] = value
+            assert model[key] == value
+
+
+def add_named_sensors_to_grafana(mac_to_name, infilename, outfilename):
+    with open(infilename) as f:
+        model = json.load(f)
+
+    mutate_grafana_model(model, mac_to_name)
+
+    with open(outfilename, "w") as f:
+        json.dump(model, f, indent=2)
+        f.write("\n")
+
+
+def add_named_sensors_to_smarthome(mac_to_name, infilename, outfilename):
+    with open(infilename) as f:
+        model = json.load(f)
+
+    mutate_smarthome_model(model, mac_to_name)
+
+    with open(outfilename, "w") as f:
+        json.dump(model, f, indent=2)
+        f.write("\n")
+
+
+if __name__ == "__main__":
+    namesfilename = os.environ.get("NAMES_INPUT", "sensor_names.conf")
+
+    with open(namesfilename) as f:
+        mac_to_name = dict(l.strip().replace(":", "").split("=") for l in f)
+
+    add_named_sensors_to_grafana(
+        mac_to_name,
+        os.environ.get("GRAFANA_INPUT", "grafana-home.json"),
+        os.environ.get("GRAFANA_OUPTUT", "new-grafana-home.json"),
+    )
+    add_named_sensors_to_smarthome(
```

Comment:
It would be useful to also fill in org.eclipse.smarthome.core.thing.link.ItemChannelLink.json, and perhaps optionally also org.eclipse.smarthome.core.items.Metadata.json.